lib/Modeler.js

```diff
@@ -1,5 +1,7 @@
 import inherits from 'inherits';
 
+import BpmnInColorModdle from 'bpmn-in-color-moddle/resources/bpmn-in-color.json';
```

Comment:
We could consider making this part of [`bpmn-moddle`](https://github.com/bpmn-io/bpmn-moddle). After all it is _defactor_ part of the standard (or standartization activities).

Reply:
Sure. This will be done via https://github.com/bpmn-io/bpmn-moddle/pull/87

Reply:
I like that idea!